cmake/CabanaConfig.cmakein

```diff
@@ -38,3 +38,7 @@ set(Cabana_ENABLE_ALL @Cabana_ENABLE_ALL@)
 if(Cabana_ENABLE_ALL)
   find_dependency(ALL @ALL_VERSION@ REQUIRED)
 endif()
+set(Cabana_ENABLE_SILO @Cabana_ENABLE_SILO@)
+if(Cabana_ENABLE_SILO)
+  find_dependency(SILO @SILO_VERSION@ REQUIRED)
```

Comment:
Do we test against an installed Cabana.  Do we need to do anything to make sure the `FindSILO.cmake` module we installed is visible?

Reply:
We have one for the base library, but not for installed dependencies. @junghans do you think we should add one for each optional library?

Reply:
We have `list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}" )` above so that should add the folder that has `FindSILO.cmake`.